src/Raven.Server/Documents/AttachmentsStorage.cs

```diff
@@ -30,38 +31,35 @@ namespace Raven.Server.Documents
 {
     public unsafe partial class AttachmentsStorage
     {
-        internal static readonly TableSchema AttachmentsSchema = Schemas.Attachments.Current;
+        internal readonly TableSchema AttachmentsSchema;
 
         private readonly DocumentDatabase _documentDatabase;
         private readonly DocumentsStorage _documentsStorage;
 
         public AttachmentsStorage(DocumentDatabase documentDatabase, Transaction tx)
         {
             _documentDatabase = documentDatabase;
+            if (_documentDatabase is ShardedDocumentDatabase)
```

Comment:
maybe it is time to split the classes completely?
Right now we have `AttachmentsStorage.cs`  & `AttachmentsStorage.Sharding.cs` which are partial of the same class.
@ppekrol thought?

Reply:
@ppekrol will do that in the future

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-19722/Sharding-Schema-tasks